src/actions.cpp

```diff
@@ -26,6 +26,7 @@
 #include "game.h"
 #include "pugicast.h"
 #include "spells.h"
+#include <fmt/format.h>
```

Comment:
Why use this in each file instead [this](https://github.com/otland/forgottenserver/pull/3430/commits/5a15e2021e4b932144acf848e283ae7e963231f1#diff-2aece0739543e6f81501e211e382a9e264966cdddecea19ed6182e61c2c0e4c5R46)?

Reply:
More includes = more compile time